src/Core/NamesAndTypes.cpp

```diff
@@ -6,7 +6,7 @@
 #include <IO/WriteHelpers.h>
 #include <IO/ReadBufferFromString.h>
 #include <IO/WriteBufferFromString.h>
-#include <sparsehash/dense_hash_map>
```

Comment:
Maybe reusing our `HashMap` (by just providing proper template parameters) will be easier.